raft/handler.go

```diff
@@ -668,24 +669,34 @@ func (pm *ProtocolManager) isVerifier(rid uint16) bool {
 	return false
 }
 
-func (pm *ProtocolManager) handleRoleChange(raftRole etcdRaft.StateType) {
+func (pm *ProtocolManager) handleRoleChange(roleC <-chan interface{}) {
```

Comment:
Why not using and `chan etcd.StateType`?

Reply:
@amalrajmani only applied the code on quorum-geth-1.9.7. we need in the future to re-do this using the right StateType API on this issue: https://github.com/ConsenSysQuorum/quorum/issues/189